tools/manifest-generator/manifest-generator.go

```diff
@@ -66,15 +82,102 @@ func main() {
 		Namespace:         *namespace,
 	}
 
-	file, err := os.OpenFile(*templFile, os.O_RDONLY, 0)
+	file, err := os.OpenFile(templFile, os.O_RDONLY, 0)
 	if err != nil {
-		glog.Fatalf("Failed to open file %s: %v\n", *templFile, err)
+		glog.Fatalf("Failed to open file %s: %v\n", templFile, err)
 	}
 	defer file.Close()
 
-	tmpl := template.Must(template.ParseFiles(*templFile))
+	tmpl := template.Must(template.ParseFiles(templFile))
 	err = tmpl.Execute(os.Stdout, data)
 	if err != nil {
 		glog.Fatalf("Error executing template: %v\n", err)
 	}
 }
+
+func generateFromCode(codeGroup string) {
+	var resources []interface{}
```

Comment:
This is to suspend type checking on the array right?  Why aren't all members of this array a Resource?

Reply:
at this layer we don't really care about the resource types (which are different for cluster or namespaced).  we just want to marshall the objects to json.  By converting to interface{} we can put cluster and namespaced resources in the same slice.